### PR TITLE
Assignment_Q3_Kunal

### PR DESCRIPTION
**Pull request by KunalKTechnocredits**
Q3. WAP to find the square and cube of a number.

### CODE REVIEW

File: JavaAssignment/src/kunal/SquareCube.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package kunal;
+public class SquareCube
+{
+	void Display(float number)
+	{
+		float Square;
+		float Cube;
+		Square = number * number;
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/SquareCube.java:8]: The way you declare the variable is not wrong, but not appropriate. The above thing simply adding multiple lines in code. 
Define directly like 
float square = number * number ;

File: JavaAssignment/src/kunal/SquareCube.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        Cube = number * number * number;
+
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/SquareCube.java:10]: REmove this space.

File: JavaAssignment/src/kunal/SquareCube.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		Square = number * number;
+        Cube = number * number * number;
+		
+        System.out.println("Square of number is : " + Square);
+        System.out.println("Cube of number is : " + Cube);
+	}
+	
+    public static void main(String[] args)
+    {
+        SquareCube sq = new SquareCube();
```
Comment (KrishnaKTechnocredits) [inline JavaAssignment/src/kunal/SquareCube.java:17]: Format your code before committing.  Press ctrl +shift + f

File: JavaAssignment/src/kunal/SquareCube.java (hunk 1 above)
Comment (mkanani13) [reply JavaAssignment/src/kunal/SquareCube.java:8]: local variable should be initialized.

**Review by mkanani13 — APPROVED** [review]
code looks good to me.

**Pull request merged**